README.md

```diff
@@ -404,6 +406,24 @@ end
 Using `insert/2` in factory definitions may lead to performance issues and bugs,
 as records will be saved unnecessarily.
 
+### Passing options to Repo.insert!/2
+
+`ExMachina.Ecto` uses
+[`Repo.insert!/2`](https://hexdocs.pm/ecto/Ecto.Repo.html#c:insert!/2) to
+insert records into the database. Sometimes you may want to pass options to deal
+with multi-tenancy or return some values generated by the database. In those
```

Comment:
How would you feel about being more explicit here about the use case for `returning`? Specifically it's not just values generated by the database, but values for all columns regardless of if those columns are specified in the factory (and by proxy, in the `insert`).

Reply:
I don't know that I want to go into full detail for what `returning` does. Rather, I was hoping to push people to look up what is available in the `Repo.insert!/2` docs. I just happened to know that multi-tenancy and returning are two things people have asked for. Do you have a suggestion on how to write this in a way that communicates that better? Maybe something like this?

> Sometimes you may want to pass options to Repo's insert function (e.g. `returning: true` or `prefix: "prefix"`). In those cases...